app/src/common/constants/screenSizeVariables.js

```diff
@@ -15,6 +15,8 @@
  */
 
 export const SCREEN_XS_MAX = 767;
+export const SCREEN_SM_MAX = 991;
+export const SCREEN_LG_MAX = 1199;
```

Comment:
Could you please rename this to `SCREEN_MD_MAX = 1199` as we have the same variable for the styles